bbcondeparser/tags.py

```diff
@@ -377,8 +377,9 @@ def walk_tree(self):
         """
         for child in self.tree:
             yield child
-            for smaller_child in child.walk_tree():
-                yield smaller_child
+            if isinstance(child, BaseTag):
```

Comment:
should this actually be this?

```suggestion
            if isinstance(child, self.__cls__):
```

Reply:
@DanElliottPalmer no, 'cause if this runs on a subclass then it excludes other descendants of BaseTag.